engine/standalone/app/src/universal/conf/application.conf

```diff
@@ -1,8 +1,10 @@
+; This config works in a simple scenario when the model provided via volumes doesn't need any configuration values
```

Comment:
Please use hash instead of semicolon. It's not windows ini file! ;) BTW this is good stuff worth to use not only in scala, take a look at full feature list: https://github.com/lightbend/config#features-of-hocon